ui_framework/components/index.js

```diff
@@ -90,6 +90,20 @@ export {
 } from './pager';
 
 export {
+  KuiTable,
+  KuiControlledTable,
+  KuiTableHeaderCell,
+  KuiTableRow,
+  KuiTableRowCell,
+  KuiTableRowCellLiner,
```

Comment:
I was hoping you'd say something like that!  Yes, if we can get away with that, I agree it makes more sense nested and less as an actual component. I wasn't quite sure what the point of it was and if it was only going to be used sometimes.  Sounds like it's used all the time for table row cells, so I'll embed it.